classes/03class/exercises/c03-docker04/jaysues/ANSWER.md

```diff
@@ -0,0 +1,26 @@
+# C03-Docker04
+
+
+## Command Execution Output
+- Commands executed to clean up both the images and the containers(running or exited) from your local:
+```
+➜  ~ docker container ls -a
+CONTAINER ID   IMAGE                          COMMAND              CREATED      STATUS      PORTS                NAMES
+17b732ad1287   jaysuesdocker/make01:5d23e6b   "httpd-foreground"   2 days ago   Up 2 days   0.0.0.0:80->80/tcp   eager_elgamal
+➜  ~ docker rm -f $(docker ps -a -q)
```

Comment:
This command will only delete all running containers, not the images. You can use 'docker image prune' command to delete all images